### PR TITLE
install all sets for CI runner

### PR DESCRIPTION
**Pull request by ZhanYF**
As mentioned in #1

### CODE REVIEW

File: resources/install.conf [hunk 1]
```diff
@@ -10,5 +10,4 @@ Which disk is the root disk = sd0
 Use (W)hole disk MBR, whole disk (G)PT or (E)dit = g
 URL to autopartitioning template for disklabel = {{ DISKLABEL_TEMPLATE }}
 Http Server = cdn.openbsd.org
-Set name(s) = -game* -x* -man*
```
Comment (jacob-carlborg) [inline resources/install.conf:13]: Are games and man pages really required? I'm trying to make the VM image as small as possible.

Comment (ZhanYF) [reply resources/install.conf:13]: It should be fine without them, if the image is only for CI runners, updated.

Comment (jacob-carlborg) [reply resources/install.conf:13]: Yes, that's the intention.

**Review by jacob-carlborg — APPROVED** [review]

**Comment by jacob-carlborg** [comment]
It adds almost 100Mb to the final image, but it's still within acceptable limits.

**Comment by ZhanYF** [comment]
Another way of doing this might be to offer different subversions, e,g: `openbsd69` for an image of full installation and `openbsd69-nox` for a smaller installation without x.
But IMHO +100Mb isn't really worth all the extra complexity because storage/bandwidth are plentiful now.

**Comment by jacob-carlborg** [comment]
> But IMHO +100Mb isn't really worth all the extra complexity

No, it's not.

> because storage/bandwidth are plentiful now.

In my opinion, storage is very limited in this case. The GitHub action runners only have a 16Gb hard drive. In this case it means the project and any files produced by this action need to fit both within the guest VM image and on the host.

**Comment by jacob-carlborg** [comment]
Thanks.

**Pull request merged**